.gitignore

```diff
@@ -1,3 +1,5 @@
-.nvmrc
 node_modules/
 dist/*
+
+
+gulp/typescript/typescript-definition-test.js
```

Comment:
Shouldn't this be included (not ignored), especially since there are changes to it below?
```suggestion
```